gluten-core/src/main/scala/io/glutenproject/execution/HashAggregateExecBaseTransformer.scala

```diff
@@ -447,7 +447,19 @@ abstract class HashAggregateExecBaseTransformer(
     var resIndex = index
     val mode = exp.mode
     val aggregateFunc = exp.aggregateFunction
-    aggregateFunc match {
+    val reIndex = aggregateFunc match {
```

Comment:
What is the difference of `reIndex` and `resIndex`? Seems to be easily mixed up.

Reply:
Done